apiserver/facades/controller/charmrevisionupdater/interface.go

```diff
@@ -37,13 +38,18 @@ type Application interface {
 type Model interface {
 	CloudName() string
 	CloudRegion() string
-	Config() (*config.Config, error)
 	IsControllerModel() bool
```

Comment:
`IsControllerModel` can be deleted as a drive by 🤔

Reply:
done